web/impact/impact/graphql/expert_profile.py

```diff
@@ -0,0 +1,29 @@
+import graphene
+from graphene_django import DjangoObjectType
+from accelerator.models.expert_profile import ExpertProfile
+from impact.graphql.user import UserType
```

Comment:
in general should we be naming type files with the suffix `_type.py`? i.e. `from impact.graphql.user_type import UserType` (mostly for convenience w/ ctrl-p)

Reply:
👍 We like file names that reflect class names.